### PR TITLE
fix: get production ready + docs

### LINKED ISSUE #59: test: execute server-side runs

### PR DESCRIPTION
**Pull request by sofinico**
Fixes and notes triggered by exercising test runs on the server.

- Switch root TS config to Node16 ESM settings and simplify per-package tsconfigs by extending the root one.
- Update committee-generator to use `.ts` import specifiers consistently and add committee-generator build-specific tsconfig.
- Improve runner behavior around 100K boundary by considering an edge case (discover this on a local test run, a race condition was taking place).
- Turn unit name as a parameter on `ExecStopPost`
- Enhance `.env.example` by providing defaults.
- Add deployment notes.

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,26 +1,39 @@
...
 S3_ACCESS_KEY_ID=
 S3_SECRET_ACCESS_KEY=
 S3_REGION=
 S3_BUCKET_NAME=
 S3_ENDPOINT=
 S3_PUBLIC_URL=
 
+# Code defaults on config.ts
+# DATA_PATH=/var/cache/xgov-committees/data
+# CONCURRENCY=5
```
[the remark this answers is not in the document]
Comment (sofinico) [reply .env.example:35]: @bwmx could you confirm which values are the best for leaving as commented default for these two variables? I just thought leaving the production defaults for the paths. LMK.

EDIT: resolved by my own criteria 😆 Just check

**Review by tasosbit — APPROVED** [review]
